.circleci/config.yml

```diff
@@ -0,0 +1,14 @@
+version: 2.1
+
+jobs:
+    Test:
+        docker:
+            - image: piperlincoln/pytest-ubuntu18.04:v0.3
```

Comment:
@gonuke do we have Svalinn or CNERG dockerhub where this could be hosted? Doesn't necessarily need to be updated for this PR, but we should move it to a group dockerhub for future support.

Reply:
We can definitely host in one of those places

Reply:
I have included my docker file in a subsequent PR I have been working on for now, but I can definitely move it to a group space!